content/webinars/getting-started-gcp-serverless-graphql/index.md

```diff
@@ -0,0 +1,91 @@
+---
+# Name of the webinar.
+title: "Getting Started with Serverless GraphQL on GCP"
+meta_desc: "In this guide, Lee Zen will walk you through building a serverless GraphQL API using Apollo GraphQL, and GCP Functions with the help of Pulumi."
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: true
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: true
+
+# The preview image will be shown on the list page.
+preview_image: "https://img.youtube.com/vi/mU5_aclyQR8/hqdefault.jpg"
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: false
+block_external_search_index: false
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "getting-started-gcp-serverless-graphql"
+
+# The content of the hero section.
+hero:
+    # The title text in the hero. This also serves as the pages H1.
+    title: "Getting Started with Serverless GraphQL on GCP"
+    # The image the appears on the right hand side of the hero.
+    image: "/icons/containers.svg"
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: "Getting Started with Serverless GraphQL on GCP"
+    # URL for embedding a URL for ungated webinars.
+    youtube_url: "https://www.youtube.com/embed/mU5_aclyQR8"
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2020-05-19T10:00:00-07:00
+    # Duration of the webinar.
+    duration: "10 minutes"
+    # Datetime of the webinar.
+    datetime: "MAY 19, 2020"
+    # Description of the webinar.
+    description: |
+        In today's episode, we are building a serverless GraphQL API using Apollo GraphQL, Google Cloud Functions, and Pulumi. Code for this episode available [here](https://github.com/pulumi/pulumitv/tree/master/modern-infrastructure-wednesday/2020-05-20).
```

Comment:
Any reason not to have this first sentence not be the same as the `meta_desc`?

```suggestion
        In this episode, Lee Zen walks you through building a serverless GraphQL API using Apollo GraphQL and GCP Functions with the help of Pulumi. The code for this episode is [available on GitHub](https://github.com/pulumi/pulumitv/tree/master/modern-infrastructure-wednesday/2020-05-20).
```

When this is rendered, though, the link doesn't show up, because the text isn't contained in a `<p>` tag:

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/274700/93390920-4c8bde00-f823-11ea-92cc-2c1874426a94.png">

We should probably fix this.

Reply:
No reason, I used the YouTube Description here but I'll fix it up to match the meta description.